<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yenimusterikayit</class>
 <widget class="QDialog" name="yenimusterikayit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>156</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>312</width>
    <height>156</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>312</width>
    <height>156</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Yeni Müşteri Kayıt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/icons/win_icon.png</normaloff>:/icons/win_icon.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>136</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Müşteri Kodu :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Müşteri Adı :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Müşteri Soyadı :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Müşteri Adresi :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="edtMusteriKod"/>
        </item>
        <item>
         <widget class="QLineEdit" name="edtMusteriAd"/>
        </item>
        <item>
         <widget class="QLineEdit" name="edtMusteriSoyad"/>
        </item>
        <item>
         <widget class="QLineEdit" name="edtMusteriAdres"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnKaydet">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Kaydet</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/icons/save_icon.png</normaloff>:/icons/save_icon.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVazgec">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Vazgeç</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/icons/close_icon.png</normaloff>:/icons/close_icon.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
